main.go

```diff
@@ -323,6 +323,11 @@ func (t *AugmentedTask) ExporterInformation() []*PrometheusTaskInfo {
 			host = ip
 		}
 
+		if host == "" {
+			// No hostname or IP found for this container. Skip.
```

Comment:
```suggestion
			// No hostname or IP found for this task. Skip.
```
only the task has an IP, not the container

Reply:
From the code perspective, The IP address is being fetched from the interfaces associated with the container. Also here we are iterating over containers associated with the task, it would be right to call it `No hostname or IP found for this container. Skip.`